releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -1,5 +1,71 @@
 # Microsoft.FeatureManagement and Microsoft.FeatureManagement.AspNetCore
 
```

Comment:
BTW, can we add a link to package(s) like other release notes, for example
https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/SpringCloudAzureFeatureManagement.md?